appdatabase/migrations/sql/1674570455_communities_permissions.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS communities_permissions (
+    community_id TEXT NOT NULL,
+    permission_id VARCHAR NOT NULL PRIMARY KEY,
+    hide BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
What does hide do?

Reply:
I think this is used to indicate that the permission is visible or not. I'd prefer to have "visible" or "public" instead of "hide".

Reply:
yes I changed it to `hidden` everywhere but not here :( are you think `visible` more correct?